### PR TITLE
 Fixes to build and test hooks for 4.3.2 release (main)

### PR DESCRIPTION
**Pull request by alanking**
Addresses https://github.com/irods/irods_rule_engine_plugin_audit_amqp/issues/139
Addresses https://github.com/irods/irods_rule_engine_plugin_audit_amqp/issues/140

Cherry-picked from https://github.com/irods/irods_rule_engine_plugin_audit_amqp/pull/141

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]
Looks identical. Pound it.

**Comment by alanking** [comment]
#'d, mergin.

File: irods_consortium_continuous_integration_build_hook.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ def install_building_dependencies(externals_directory):
     externals_list = [
         'irods-externals-avro1.11.0-3',
         'irods-externals-boost1.81.0-1',
+        'irods-externals-clang-runtime13.0.1-0',
```
Comment (SwooshyCueb) [inline irods_consortium_continuous_integration_build_hook.py:21]: This branch doesn't need `clang-runtime`, as it's build against libstdc++

Comment (alanking) [reply irods_consortium_continuous_integration_build_hook.py:21]: Good call. Created https://github.com/irods/irods_rule_engine_plugin_audit_amqp/issues/148 for this.

**Pull request merged**